package.json

```diff
@@ -30,9 +30,11 @@
   "dependencies": {
     "async": "^0.9.x",
     "jsonld": "^0.4.5",
+    "mz": "^2.6.0",
```

Comment:
I'm using mz for the promisified version of the load and write. We could promisify fs-extra by:
```
const Promise = require('bluebird')
const fs = Promise.promisifyAll(require('fs-extra'))
```
(which I think was probably the reason why I started to use bluebird promises in the first place)